src/main/java/com/kelinci/mainapp/api/UserController.java

```diff
@@ -9,24 +9,18 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
 @RestController
 public class UserController {
-    private OurUser registeredUser;
-    private OurUser confirmedUser;
+    private OurUser ourUser;
 
     private List<OurUser> listOfUsers = new ArrayList<>();
```

Comment:
Mnie tu IntelliJ podkreśla 'listOfUser' na sraczkowo żółto. A Tobie? Też? To co tam Ci mówi Intellij? :)

Reply:
żeby zrobić final - ok można faktycznie